### ISSUE TITLE
Some plots are not rendered in QWebView (Scatter Plot 3D, Contour, Heatmap)

### ISSUE THREAD

**Issue by ghtmtt** [ghtmtt/DataPlotly#13]
Apparently the code works (dict is filled with correct values etc), the base plot (grid, axes, etc) is visible in the QWebView, but **no points or lines appear in the plot**.

This happens with the following:
* ScatterPlot 3D
* Heatmap
* Contour

Not working solution it to locally save the `plotly.js` file and load it in the html file. In this case nothing is loaded in the `QWebView`

**Comment by ghtmtt** [comment]
it seems that the problem is of the `QWebView` widget. static html files can be opened with a browser and are correctly rendered.
By adding these rows in the `init` function it is possible to open the debug mode and see the error (not fixed yet):

```
        plot_view_settings = self.plot_view.settings()
        plot_view_settings.setAttribute(QWebSettings.WebGLEnabled, True)
        plot_view_settings.setAttribute(QWebSettings.DeveloperExtrasEnabled, True)
```

**Comment by ghtmtt** [comment]
3d plot is fixed in 9182d54f498e607fdda7783296f075026ae43e0f
while for both Heatmap and Contour, they actually work, but the input data needs to by subdivided into a matrix. 
By pasting the code from [here](https://plot.ly/python/contour-plots/) in the plugin, plots are visible. So it is an input data stuff to solve.
